woocommerce/woocommerce-ios
# Order Status Mark 1: Add new entity and change attribute name

Changed the OrderStatus enum to OrderStatusKey. Added the new entity OrderStatus (but it's not used or mapped anywhere yet, in Networking). This PR is mostly generated code.

## To Test
1. Delete app, reset Content & Settings
2. Check out PR and build
3. Run unit tests and ensure they're green

cc: @astralbodies and @bummytime ready for review!

Update release notes:
